rules/fw_rules.yaml

```diff
@@ -0,0 +1,40 @@
+rules:
+  - rule_id: 'prevent_allow_all'
+    description: Prevent allow all policies
+    type: blacklist
+    match_policies:
+      - direction: ingress
+        allowed: ['*']
+    verify_policies:
+      - allowed:
+        - IPProtocol: 'all'
+
+  - rule_id: 'disallow_any_port'
+    description: Don't allow policies that expose every port
+    type: blacklist
+    match_policies:
+      - direction: ingress
+        allowed: ['*']
+    verify_policies:
+      - allowed:
+        - IPProtocol: 'tcp'
+          ports:
+            - 'all'
+      - allowed:
+        - IPProtocol: 'udp'
+          ports:
+            - 'all'
+
+rule_groups:
+  - group_id: 'default_rules'
+    rule_ids:
+      - 'prevent_allow_all'
+        'disallow_any_port'
+
+org_policy:
+  resources:
+    - type: organization
+      resource_ids:
```

Comment:
nit: Can you please put a placeholder, like ```YOUR_ORG_ID / YOUR_RESOURCE_ID```  Can the user also just use a wildcard? ```*```

Also, can this be directed toward projects & project ids?

Reply:
| Also, can this be directed toward projects & project ids?
Yes it can. Since this will be the default rule in place I wanted it to be as simple and easy to set up as possible, so this will be the one line that our users will have to change. In the samples there will be more complicated examples of the hierarchy.

| Can the user also just use a wildcard? *
These policies are hierarchy aware, so our users will need to configure them for that. For example, if they want a policy to everything, then they can set it at the org level.